### PR TITLE
Update configs

### PR DESCRIPTION
**Pull request by X9VoiD**
And also lint the config files using `jq`.

### CODE REVIEW

**Review by gonX — APPROVED** [review]
We should probably add `jq` as a linting step - making an issue

File: TABLETS.md [hunk 1]
```diff
@@ -2,18 +2,23 @@
...
 | Gaomon M106K Pro              |     Supported     |
 | Gaomon PD1161                 |     Supported     |
 | Gaomon PD1560                 |     Supported     |
 | Gaomon PD1561                 |     Supported     |
-| Gaomon S56K                   |     Supported     |
+| Gaomon PD2200                 |     Supported     |
+| Gaomon S56K                   |     Supported     | Windows: Some variations may require Zadig's WinUSB to be installed on interface 0 or 1
 | Gaomon S620                   |     Supported     |
 | Gaomon S630                   |     Supported     | Windows: Requires Zadig's WinUSB to be installed on interface 0
+| Gaomon S830                   |     Supported     | User may need to re-plug their tablet multiple time for it to be detected
```
Comment (Sublimelime) [inline TABLETS.md:14]:
```suggestion
| Gaomon S830                   |     Supported     | User may need to re-plug their tablet multiple times for it to be detected
```

File: TABLETS.md [hunk 2]
```diff
@@ -2,18 +2,23 @@
...
 | Genius G-Pen 560              |     Supported     | Soft-buttons are bindable as aux buttons
-| Genius i608x                  |     Supported     | Require Zadig's WinUSB
+| Genius i405x                  |     Supported     | Windows: Require Zadig's WinUSB
+| Genius i608x                  |     Supported     | Windows: Require Zadig's WinUSB
```
Comment (Sublimelime) [inline TABLETS.md:17]: Interface? Also Require -> Requires

File: TABLETS.md [hunk 3]
```diff
@@ -22,40 +27,49 @@
...
 | Huion Kamvas Pro 13 (2.5k)    |     Supported     |
+| Huion Kamvas Pro 16 (2.5k)    |     Supported     |
 | Huion New 1060 Plus           |     Supported     |
 | Huion New 1060 Plus (2048)    |     Supported     | Windows: Requires Zadig's WinUSB to be installed on interface 1
 | Huion osu! Tablet             |     Supported     | Windows: Requires Zadig's WinUSB to be installed on interface 0
 | Huion Q11K                    |     Supported     |
 | Huion Q11K V2                 |     Supported     |
 | Huion WH1409                  |     Supported     |
 | Huion WH1409 V2               |     Supported     |
+| KENTING K5540                 |     Supported     | Windows: Requires Zadig's WinUSB
```
Comment (Sublimelime) [inline TABLETS.md:50]: Interface?

File: TABLETS.md [hunk 4]
```diff
@@ -126,27 +142,34 @@
...
 | Huion Kamvas Pro 24           |  Missing Features | Touch bar is not yet supported.
 | Huion Q620M                   |  Missing Features | Wheel is not yet supported.
+| Huion RTM-500                 |  Missing Features | Touch bar is not yet supported.
 | Parblo A610 Pro               |  Missing Features | Wheel is not yet supported.
 | Parblo Intangbo M             |  Missing Features | Wheel is not yet supported.
 | Parblo Intangbo S             |  Missing Features | Wheel is not yet supported.
 | UGEE EX08                     |  Missing Features | Tilt is not yet supported. Uses the same configuration as the XP-Pen Deco 01 V2.
 | VEIKK A15 Pro                 |  Missing Features | Wheel is not yet supported.
 | VEIKK A30                     |  Missing Features | Touchpad is not yet supported.
+| Veikk A30 V2                  |  Missing Features | Touchpad is not yet supported.
```
Comment (Sublimelime) [inline TABLETS.md:162]:
```suggestion
| VEIKK A30 V2                  |  Missing Features | Touchpad is not yet supported.
```

**Review by Sublimelime — APPROVED** [review]
LGTM Except for some minor grammar things in Tablets.md.

File: TABLETS.md (hunk 3 above)
Comment (jamesbt365) [reply TABLETS.md:50]: Its a singular interface device.

File: TABLETS.md (hunk 2 above)
Comment (jamesbt365) [reply TABLETS.md:17]: Singular interface devices

File: TABLETS.md (hunk 4 above)
Comment (jamesbt365) [reply TABLETS.md:162]: Blame snowlire

File: TABLETS.md (hunk 2 above)
Comment (InfinityGhost) [inline TABLETS.md:17]:
```suggestion
| Genius i405x                  |     Supported     | Windows: Requires Zadig's WinUSB
| Genius i608x                  |     Supported     | Windows: Requires Zadig's WinUSB
```

**Review by InfinityGhost — CHANGES_REQUESTED** [review]
~~Multiple configurations have final newlines and extra whitespace.~~ Forgot to pull lmao

**Review by InfinityGhost — APPROVED** [review]

**Pull request merged**